build-all.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I could not figure out how to do this. Since we are only using this to build images, I think it ok to use bash instead of sh in this case.